schemas/context/solutionidentifiers.schema.json

```diff
@@ -0,0 +1,53 @@
+{
+  "meta:license": [
+    "Copyright 2017 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/context/solutionidentifiers",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Solution identifiers",
+  "type": "object",
+  "meta:extensible": true,
+  "description": "Condensed, normalized encapsulation of solution identifiers.",
+  "definitions": {
+    "solutionidentifiers": {
+      "properties": {
+        "https://ns.adobe.com/experience/aaid": {
```

Comment:
See earlier comments about proper namespacing

Reply:
Yes, but this needs to be a solution identifier under experience namespace

Reply:
the changes have been made